src/schneider_lab_to_nwb/zempolich_2024/zempolich_2024_metadata.yaml

```diff
@@ -25,6 +25,62 @@ BrainRegion:
     electrode_group_location: Secondary Motor Cortex (M2)
     optogenetic_stimulus_site_location: Secondary Motor Cortex (1.0-1.5 AP, 0.5-0.7 ML)
 
+SubjectMaps:
```

Comment:
Do you lad this as editable metadata? I am surprised this does not throw a validation error. 

I have a slight feeling that this would be better in a separate file but your mileage might vary. 

Rest looks good to me.

Reply:
> I am surprised this does not throw a validation error.

Maybe this is a bug, but the validator doesn't check any entries that aren't in the schema at the top level.  I could put anything I want in SubjectMaps and it would pass bc it's defined at the top level of the metadata.

> I have a slight feeling that this would be better in a separate file but your mileage might vary.

I just like to keep all of the metadata in one place (if it's not already pre-defined elsewhere) to make it easier to edit by the lab.